index.html

```diff
@@ -51,5 +51,116 @@ <h1 class="intro-header">I’m Jordan Glad to see you!</h1>
+            <div class="features">
+                <div class="f f1 upper-case">canopy</div>
+                <span>.</span>
+                <div class="f f2">Back End Dev</div>
+                <span>.</span>
+                <div class="f f3">2015</div>
+            </div>
+            <p class="p p-proj">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="tags">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>                
+            </ul>
+            <a class="btn sp-button" href="#">See Project</a>            
+        </div>
+    </section>
+
+    <section id="section-3" class="padding">
+        <h2 class="header intro-header">About <br>Myself</h2>
+            <p class="p intro-p">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p class="upper-case lets-connect">let’s connect</p>
+            <ul class="social-media-icons">
+                <li class="icon github-icon"><a href="https://github.com/Jord4N-AC"><img src="images/icons/github_icon.svg" alt="Github Icon"></a></li>
+                <li class="icon linkedin-icon">
+                    <a href="https://www.linkedin.com/in/camilo-armijos-2b9648197/">
+                        <div class="i-container">
+                            <img class="i1" src="images/icons/linkedin1.svg" alt>
+                            <img class="i2" src="images/icons/linkedin2.svg" alt>
+                        </div> 
+                        <img class="n" src="images/icons/linkedin3.svg" alt="LinkedIn Icon">
+                    </a>
+                </li>
+                <li class="icon hand-icon"><a href="https://www.angellist.com/"><img src="images/icons/hand_icon.svg" alt="Hand Icon"></a></li>
+                <li class="icon twitter-icon"><a href="https://twitter.com/jordan12AC"><img src="images/icons/twitter_icon.svg" alt="Twitter Icon"></a></li>
+                <li class="icon m-icon"><a href="https://medium.com/"><img src="images/icons/m_icon.svg" alt="M Icon"></a></li>
+            </ul>
+            <a class="btn gmr-button" href="#">Get my resume</a>
+            <ul id="skills">
+                <li>
+                    Langguages
+                    <span>&#62;</span>
+                </li>                                                                           
+                <li>
+                    Frameworks
+                    <span>&#62;</span>
+                </li>                                                                                                
+                <li>
+                    Skills
+                    <span>&#62;</span>
+                </li>                                                               
+            </ul>         
```

Comment:
[OPTIONAL]

- [ ] Consider making your work match that of your chosen template 😃 .

Your design:

<img width="144" alt="chrome_N1iZtriZsY" src="https://user-images.githubusercontent.com/81468675/143722355-729b84e0-1294-4518-bdf2-298a56607c3b.png">

Figma design: 
<img width="158" alt="chrome_dAnLT36hLe" src="https://user-images.githubusercontent.com/81468675/143722362-70af78c7-6d7c-4a8d-bf37-701a5310c608.png">